.github/workflows/jira-issue-create.yml

```diff
@@ -29,11 +29,9 @@ jobs:
           description: |
             ${{ github.event.issue.html_url }}
           fields: '{
-            "customfield_10006": 1391,
-            "customfield_11481": {"value": "Governance 🕵️"},
-            "customfield_11200": {"value": "Developer Experience"},
+            "customfield_11794": "Github",
             "labels": ["ts"]
-          }' # sprint, pillar, pod, labels
+          }' # origin, labels
```

Comment:
It will happen here: https://github.com/amplitude/Amplitude-TypeScript/pull/162/files#diff-02104d32f0f9edecea1491f3d89fe2214a1c824f0ca2b186af6c3c7bfc8ef25cR25 I added a release notes in this PR to update the github secrets.

Reply:
Should the project key be `DXOC` for the oncall board?

Reply:
Yes